documentation/docs/user_guide/inputs/cable_nml.md

```diff
@@ -83,74 +83,127 @@ applications. The following are annotated examples of cable.nml:
-| wiltParam                    | real      |                                     | 0.5                                                                          | Wilt parameter in hydraulic redistribution module.                                                       |
-| satuParam                    | real      |                                     | 0.8                                                                          | Saturation parameter in hydraulic redistribution module.                                                 |
-| cable_user%FWSOIL_SWITCH     | character | 'standard' 'non-linear extrapolation' 'Lai and Ktaul 2000' | 'standard'                                            | Controls root water uptake function.                                                                     |
-| cable_user%DIAG_SOIL_RESP    | character(len=3) | 'ON ' 'OFF'                  | 'ON '                                                                        | Controls soil respiration scheme when CASA-CNP not used.                                                 |
-| cable_user%LEAF_RESPIRATION  | character(len=3) | 'ON ' 'OFF'                  | 'ON '                                                                        | Controls what is output into the GPP stash variable.                                                     |
-| cable_user%RUN_DIAG_LEVEL    | character | 'BASIC' 'NONE'                      | 'BASIC'                                                                      | Controls output from CABLE to standard out.                                                              |
-| cable_user%CONSISTENCY_CHECK | logical   | .TRUE. .FALSE.                      | .TRUE.                                                                       | TRUE outputs combined fluxes at each timestep for comparison to a control run                            |
-| cable_user%CASA_DUMP_READ    | logical   | .TRUE. .FALSE.                      | .FALSE.                                                                      | TRUE reads CASA forcing from netcdf format                                                               |
-| cable_user%CASA_DUMP_WRITE   | logical   | .TRUE. .FALSE.                      | .FALSE.                                                                      | TRUE outputs CASA forcing in netcdf format                                                               |
-| cable_user%SSNOW_POTEV       | character(len=3) | 'HDM' 'P-M'                  | 'HDM'                                                                        | Use the "Humidity Deficit Method" or  "Penman-Monteith" method                                           |
+| Namelist variable                | Type               | Available values                                           | Default value                    | Description                                                                                             |
```

Comment:
```suggestion
## List of namelist variables

| Namelist variable                | Type               | Available values                                           | Default value                    | Description                                                                                             |
```

Reply:
This appears weird here but correct in the review panel. This should simply add a section title before the table.